.github/workflows/ci.yml

```
@@ -10,57 +10,138 @@ on:
   pull_request:
   workflow_dispatch:
 
+env:
+  IMAGE_NAME: openformulieren/open-forms-ext-stuf-zds-payments
+
 jobs:
   tests:
+    name: Run the Django test suite
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        python: ['3.10', '3.11', '3.12']
-        django: ['4.2']
-
-    name: Run the test suite (Python ${{ matrix.python }}, Django ${{ matrix.django }})
+    services:
+      postgres:
+        image: postgres:14
+        env:
+          POSTGRES_HOST_AUTH_METHOD: trust
+        ports:
+          - 5432:5432
+        # Needed because the postgres container does not provide a healthcheck
+        options:
+          --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+          --name postgres
+      redis:
+        image: redis:6
+        ports:
+          - 6379:6379
 
     steps:
-      - uses: actions/checkout@v4
-      - uses: actions/setup-python@v5
+      - name: Checkout Open Forms
+        uses: actions/checkout@v3
```

```suggestion
        uses: actions/checkout@v4
```
v3 uses deprecated node version, see the warnings here: https://github.com/open-formulieren/open-forms-ext-stuf-zds-payments/actions/runs/10144521175?pr=1